src/timesync/timesyncd-server.h

```diff
@@ -12,6 +12,7 @@ typedef enum ServerType {
         SERVER_FALLBACK,
         SERVER_LINK,
         SERVER_RUNTIME,
+        _SERVER_TYPE_MAX,
```

Comment:
while you are at it, maybe also add:

```c
_SERVER_TYPE_INVALID = -EINVAL,
```

we typically have that all across our codebase, let's do thta here too

Reply:
done